lua/bufferline/utils.lua

```diff
@@ -76,7 +76,7 @@ return {
   hl = {
     --- @class barbar.utils.hl.group
     --- @field cterm integer|string
-    --- @field gui integer|string
```

Comment:
`gui` can never be an `integer`— try `:hi Foo guifg=444` (etc), it doesn't work.